src/ripple/app/paths/cursor/DeliverNodeReverse.cpp

```diff
@@ -251,7 +252,14 @@ TER PathCursor::deliverNodeReverseImpl (
             resultCode = increment(-1).deliverNodeReverseImpl(
                 node().offerOwnerAccount_,
                 saInPassReq,
-                saInPassAct);
+                saInPassAct,
+                saOutAct > zero);
+
+            if (resultCode == tecPATH_DRY && saOutAct > zero)
+            {
+                resultCode = tesSUCCESS;
```

Comment:
Why can't `deliverNodeReverseImpl` itself convert the `tecPATH_DRY` to a `tesSUCCESS` as appropriate? Are the circumstances where we don't want this behavior?